### ISSUE TITLE
Multilingual

### ISSUE THREAD

**Issue by jfreak53** [danielgtaylor/aglio#125]
I'm looking in the documentation but not finding it. Is there a way to support the internal items in other languages? I would like some of the terms in spanish:
http://cdn.microthosting.com/imgs/Screenshot_at_2015-06-24_14-18-04.png

Like `show` and the other internal terms. I can translate just need to know where and how.

**Comment by zdne** [comment]
Hey @jfreak53 as far as I know now there is not internationalization in the Aglio at the moment, however let's wait for @danielgtaylor to confirm.

**Comment by danielgtaylor** [comment]
This is a duplicate of #7. Please continue discussion there, and I'll close this issue for now. Short answer: no internationalization at the moment. I'm open to the idea, but it should have a good plan in place before we start coding.
